components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -172,7 +172,7 @@ class GeckoEngineSession(
         }
 
         geckoSession.load(loader)
-        Fact(Component.BROWSER_ENGINE_GECKO_BETA, Action.IMPLEMENTATION_DETAIL, "GeckoSession.load").collect()
```

Comment:
This looks wrong. 🤔

Reply:
@mcomella is this something that was supposed to stay with the current variant?

Reply:
I can fix this for now to unblock the release process.

Reply:
Are these generated files? This looks like it was implemented correctly in https://github.com/mozilla-mobile/android-components/pull/9601/ so I'm not sure why this was incorrect on master. Your corrections look good now though.

Reply:
Yeah, that's correct. During merge day we copy over the source from beta -> release and nightly -> beta. So I guess this happened during the previous merge day unknowingly.

We can probably do better by making the Component name somewhat generic `Component.BROWSER_ENGINE` and picking the right one based on the variant.

You can read the process [here](https://mozac.org/contributing/merge-day).

Reply:
Thanks for the info – are there any use cases where these files would be edited manually? If not, I wonder if it'd be useful to add a check to prevent it from happening, e.g. a pre-push hook. I'm concerned that a change might go missing or would be incorrectly manipulated (like in this case) though I don't fully understand the system to know if this is really a problem or not. Seems like it can get caught here at least. 🙂

Reply:
Beta and release should be expected to be overwritten by nightly on merge day. The problem here is that we have something in those variants which should always stay in that variant, so we should try to find a better way to reference the `Component` class name.

I've filed this issue to continue the discussion there: https://github.com/mozilla-mobile/android-components/issues/9908